sklearn/covariance/empirical_covariance_.py

```diff
@@ -69,7 +69,8 @@ def empirical_covariance(X, assume_centered=False):
     """
     X = np.asarray(X)
     if X.ndim == 1:
-        X = np.reshape(X, (1, -1))
+        X = X[:, np.newaxis]
```

Comment:
Why do you reshape it the other way around now?
And shouldn't we just use check_array here and raise the deprecation warning?